root.tf

```diff
@@ -484,6 +493,36 @@ module "export_step_function" {
   })
 }
 
+module "new_export_bucket" {
+  source            = "./da-terraform-modules/s3"
+  bucket_name       = "tdr-export-${local.environment}"
```

Comment:
The bucket name should be suffixed with a random id.

Reply:
I think I've updated this since you wrote it but the module now has a `use_random_suffix` variable that adds it on to the name. It's there to stop breaking existing buckets. I've set it to true on these two so they'll get the random name.